subprojects/diagnostics/src/main/java/org/gradle/api/plugins/HelpTasksPlugin.java

```diff
@@ -90,6 +100,11 @@ public void apply(final Project project) {
             task.getShowAll().convention(false);
             task.getRecursive().convention(false);
         });
+        tasks.register(ARTIFACT_TRANSFORMS_TASK, ArtifactTransformsReportTask.class, task -> {
+            task.setDescription("Displays the Artifact Transforms that can be resolved in " + projectName + ".");
```

Comment:
```suggestion
            task.setDescription("Displays the Artifact Transforms that can be executed in " + projectName + ".");
```

Nitpick, but I'd say transforms are _executed_ or maybe even _selected_ instead of _resolved_